### ISSUE TITLE
syntax error near unexpected token `)

### ISSUE THREAD

**Issue by joshuacox** [bevry/dorothy#228]
during `setup-system install`  I have run into this sort of corrupted setup.bash file.

```
┌  Setup Node.js  ┐
The [node] utility is marked for upgrade. Upgrading... ⏲
┌  Install Node.js via System  ┐
Installed: v22.2.0
└  Install Node.js via System  ┘
┌  Clean Node.js versions and packages  ┐
Now using system version of node: v22.2.0 (npm v10.8.1)
Uninstalled: v20.14.0
nvm: Cannot uninstall currently-active node version, v22.2.0 (inferred from 22.2.0).
└  Clean Node.js versions and packages  ┘
┌  Clean Node.js Version Manager (NVM) instead  ┐
└  Clean Node.js Version Manager (NVM) instead  ┘
┌  Install Node.js Version Manager (NVM)  ┐
NVM_DIR=/home/thoth/.nvm
└  Install Node.js Version Manager (NVM)  ┘
┌  Install Node.js via NVM  ┐
Installs active and LTS versions.
Installed: v20.14.0
Installed: v22.2.0
└  Install Node.js via NVM  ┘
┌  Upgrade npm  ┐
Before: v10.7.0
After:  v10.8.1
└  Upgrade npm  ┘
The [node] utility was upgraded via [eval]
The [node] utility was upgraded. ✅
┌  Configure npm  ┐
└  Configure npm  ┘
The [gopls] utility is already installed. ✅
The [Delve] utility is already installed. ✅
The [golangci-lint] utility is already installed. ✅
└  Setup Go  ┘
/home/user1/.local/share/dorothy/user/config/setup.bash: line 1: syntax error near unexpected token `)'
```

looking in this file is some truncated half dead file:

```
)
GO_LINTING_INSTALL='yes'
```

removing it, and re-running

 `setup-system install ` 

nets the same results, and the above strange setup.bash file is recreated in its half dead state.

**Comment by balupton** [comment]
Thank you for the report.

Can you attach a copy of your setup.bash file or email it to b@bevry.me if it is private, as there likely is something in there that https://github.com/bevry/dorothy/blob/master/commands/dorothy-config is struggling with.

**Comment by joshuacox** [comment]
@balupton unfortunately the entirety of the file was what I had posted earlier

```
)
GO_LINTING_INSTALL='yes'
```

and while repeatable at the time, I just began doing all the individual jobs like `setup-utils` etc individually.  I can now do the install successfully without it failiing (*with the exception that I cannot seem to include ksh without it freezing, but that is perphaps  a separate issue).  Unfortunately, I am unable to nail down the exact cause.  I am going to close for now, I will re-open if I can uncover any more information.

**Comment by balupton** [comment]
So this is the same issue as https://github.com/bevry/dorothy/issues/241 which has been solved by https://github.com/bevry/dorothy/issues/241

**Comment by balupton** [comment]
I have finally been able to identify what seems to have been the cause of this issue, and it will soon be resolved.

**Comment by balupton** [comment]
Closing as fixed within #281, which is now ready for merging, and will be merged shortly.
